lxqt-config-brightness/monitorinfo.cpp

```diff
@@ -19,10 +19,12 @@
 #include "monitorinfo.h"
 
 MonitorInfo::MonitorInfo(int id, QString name, long backlightMax)
+    : mBacklightMax(backlightMax),
+      mBacklight(-1),
+      mBrightness(-1.0f),
```

Comment:
Shouldn't this be `1.0f`? `mBacklight` can be negative ("negative" means "not supported") but `mBrightness` is between `0` and `2`.

Reply:
My rationale was to initialize `mBrightness` with an impossible value to distinguish it from the ones set with `setBrightness()`

Reply:
I didn't mean that it's a real problem; the PR is good. IMO, "1.0f" for brightness (=100% brightness) is just a little more logical as the default.

Anyway: GTM.

Reply:
> I didn't mean that it's a real problem; the PR is good. IMO, "1.0f" for brightness (=100% brightness) is just a little more logical as the default.
> 
> Anyway: GTM.

I do understand your point. Since the correct/current value is unknown setting it to a possible value seems odd to me.
Maybe the range is 0.0f to 2.0f to accommodate gamma when backlight is not available.